### ISSUE TITLE
Dutch stopwords

### ISSUE THREAD

**Issue by marcselman** [MihaiValentin/lunr-languages#7]
Hi, I've been trying out your great Lunr extension.
It seems to work for the most part except that dutch stopwords (in my case "en") do not seem to be ignored.
Do you have any idea what the problem might be?

Thank you

**Comment by olivernn** [comment]
The problem is where the [list of stop words are declared](https://github.com/MihaiValentin/lunr-languages/blob/master/lunr.du.js#L433)

For reasons that I can't remember now the stopWordFilter uses a lunr.SortedSet as the datastructure for keeping track of the stop words. A key part of the SortedSet is that the set of values are sorted, the SortedSet usually maintains this characteristic, but they way the list of words are being loaded bypasses this check, setting `SortedSet#elements` directly.

Because the list is not sorted the `SortedSet#indexOf` check fails to correctly find the word "en" (and probably others also).

The quick fix for this is to always sort the list of stopWords, or ensure they are already sorted in the source file, see the [english stop word filter](https://github.com/olivernn/lunr.js/blob/master/lib/stop_word_filter.js#L24-L145) for example.

The SortedSet could be entirely replaced by a JavaScript object with the stopWords as properties, being careful to not confuse other, inherited properties of a JavaScript object as stopWords (e.g. "valueOf")

A better, long term, fix would be for lunr to provide an easier way of creating custom stopWordLists, something with an API like this:

    dutchStopWords = lunr.stopWordFilter('de en van ik te dat die in een hij het niet')

**Comment by marcselman** [comment]
I understand. I have changed the stopwords part in the dutch source file:

```
	// Looking at the original English stoplist the length should be equal to the length of the array minus 1.
	// I guess the empty element at the beginning of the array is not counted.
	lunr.du.stopWordFilter.stopWords.length = 101;
	lunr.du.stopWordFilter.stopWords.elements = [
		'',
		'aan',
		'al',
		'alles',
		'als',
		'altijd',
		'andere',
		'ben',
		'bij',
		'daar',
		'dan',
		'dat',
		'de',
		'der',
		'deze',
		'die',
		'dit',
		'doch',
		'doen',
		'door',
		'dus',
		'een',
		'eens',
		'en',
		'er',
		'ge',
		'geen',
		'geweest',
		'haar',
		'had',
		'heb',
		'hebben',
		'heeft',
		'hem',
		'het',
		'hier',
		'hij',
		'hoe',
		'hun',
		'iemand',
		'iets',
		'ik',
		'in',
		'is',
		'ja',
		'je',
		'kan',
		'kon',
		'kunnen',
		'maar',
		'me',
		'meer',
		'men',
		'met',
		'mij',
		'mijn',
		'moet',
		'na',
		'naar',
		'niet',
		'niets',
		'nog',
		'nu',
		'of',
		'om',
		'omdat',
		'onder',
		'ons',
		'ook',
		'op',
		'over',
		'reeds',
		'te',
		'tegen',
		'toch',
		'toen',
		'tot',
		'u',
		'uit',
		'uw',
		'van',
		'veel',
		'voor',
		'want',
		'waren',
		'was',
		'wat',
		'werd',
		'wezen',
		'wie',
		'wil',
		'worden',
		'wordt',
		'zal',
		'ze',
		'zelf',
		'zich',
		'zij',
		'zijn',
		'zo',
		'zonder',
		'zou'
    ];
```

This seems to work correctly. Maybe you can update your source files.

Thank you.

**Comment by MihaiValentin** [comment]
Hi @marcselman ,

Thanks for pointing out the issue. I continued the discussion regarding this on the #8 and will fix as soon as possible.

Thanks!

**Comment by MihaiValentin** [comment]
Fixed! Check #8 for more details.
